### DISCUSSION TITLE
Where to start?

### DISCUSSION THREAD

**Discussion by ylnest2018** [iptv-org/community#1119] [Q&A]
I’m looking for a couple channels, they exist in the channel list csv, I’ve used traditional m3u playlists, wondering where I start to be able to stream one of those channels?

**Comment by thomraider12** [comment]
You can go into their website here <https://github.com/iptv-org/iptv/tree/master/streams> and choose an stream, according to your country/SamsungTV country.

**Reply by ylnest2018** [reply]
thanks I will take a look at the link you shared, I used the M3U playlist: http://iptv-org.github.io/iptv/index.m3u

cant seem to find these:

RaiItaliaAfrica.it
RaiItaliaAmerica.it
RaiItaliaAsia.it
RaiItaliaAustralia.it

**Reply by thomraider12** [reply]
You can find the America one and the Asia one.

**Reply by ylnest2018** [reply]
Neither function though

**Reply by thomraider12** [reply]
For me they do. Check on another player.

**Comment by ylnest2018** [comment]
Which player are you using? They’re not geoblocked are they?

I’ve tried 4-5 different players on both iOS and macOS

**Reply by thomraider12** [reply]
IPTV Pro. No they aren't.

**Reply by ylnest2018** [reply]
Do you have a link? What platform? Everything in App Store is called IPTV Pro 😂

**Reply by thomraider12** [reply]
> Do you have a link? What platform? Everything in App Store is called IPTV Pro 😂

It's only in the Google Play Store. It's paid, but it works. It's that only named app there.

**Reply by ylnest2018** [reply]
I have tried everything, all receive errors. Nearly all other channels work so I can't be doing something wrong here. Any other suggestions, it is literally the only channel I require

**Comment by ylnest2018** [comment]
![image](https://github.com/user-attachments/assets/3d95a6c1-d55d-4d61-b95c-ced440cc5725)

**Comment by thomraider12** [comment]
Ok, I don't know then.

A quinta, 26/09/2024, 06:40, ylnest2018 ***@***.***> escreveu:

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/master/FAQ.md#why-is-the-channel-on-the-iptv-orggithubio-but-not-in-the-playlist
